src/StartJourneyButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './App.css';
+
+type Props = {
+    gotStarted: boolean,
+    setGotStarted: Function
+};
```

Comment:
Also, you can use an interface
```suggestion
interface StartJourneyButtonProps {
    gotStarted: boolean,
    setGotStarted: Function
};
```